Chutzpah/Models/ChutzpahTestSettingsFile.cs

```diff
@@ -350,35 +356,40 @@ public ChutzpahTestSettingsFile InheritFrom(ChutzpahTestSettingsFile parent)
             this.CodeCoverageIgnores = parent.CodeCoverageIgnores.Concat(this.CodeCoverageIgnores).ToList();
             this.Transforms = parent.Transforms.Concat(this.Transforms).ToList();
 
+            foreach (var browserArgument in parent.BrowserArguments)
+            {
+                this.BrowserArguments[browserArgument.Key] = browserArgument.Value;

```

Comment:
This doesn't look right. We should only override the child if the child does not already define a value for a key